playground/mocks/data/idp/idx/authenticator-verification-phone-sms-nickname.json

```diff
@@ -0,0 +1,312 @@
+            "visible":false,
+            "mutable":false
+          }
+        ]
+      },
+      "type":"phone",
+      "key": "phone_number",
+      "id":"id-phone-authenticator",
+      "displayName":"id-phone-authenticator",
+      "methods":[
+        {
+          "type":"sms"
+        }
+      ]
+    }
+  },
+  "authenticatorEnrollments": {
+    "type": "array",
+    "value": [
+      {
+        "displayName": "Okta Password",
+        "type": "password",
+        "key": "okta_password",
+        "id": "autwa6eD9o02iBbtv0g1",
+        "authenticatorId": "aidwboITrg4b4yAYd0g3"
+      },
+      {
+        "displayName": "FIDO2 (WebAuthn)",
+        "type": "security_key",
+        "key": "webauthn",
+        "id": "autwa6eD9o02iBbtv0g2",
+        "authenticatorId": "aidtheidkwh282hv8g3"
+      },
+      {
+        "displayName": "FIDO2 (WebAuthn)",
+        "type": "security_key",
+        "key": "webauthn",
+        "id": "autwa6eD9o02iBbtv0g2",
+        "authenticatorId": "fwftheidkwh282hv8g3"
+      },
+      {
+        "displayName": "Okta Email",
+        "type": "email",
+        "key": "okta_email",
+        "authenticatorId": "aidtm56L8gXXHI1SD0g3",
+        "id": "autwa6eD9o02iBbtv0g3",
+        "methods": [
+          {
+            "methodType": "email"
+          }
+        ]
+      },
+      {
+        "displayName": "Okta Phone",
+        "nickname": "phone-nn",
```

Comment:
This doesn't seem to be in the correct location, if you look at [this](https://github.com/okta/okta-signin-widget/pull/3342/files#diff-9396975b565d396e86d9d556a15dc53d090538a44eec282d8b7c86c0df673560R14-R16) it is pointing to the currentAuthenticatorEnrollment, which does not have a nickname set. So the test will fail because the nickname wont be available in the view.

Reply:
yes good catch. Thanks

Reply:
Moved to `authenticatorEnrollments`. Thanks